### PR TITLE
Add missing database options in class icingadb

### PR DESCRIPTION
**Pull request by lbetz**
refs #35 

Options are described in
https://icinga.com/docs/icinga-db/latest/doc/03-Configuration/#database-options.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -145,6 +149,11 @@
   Optional[Stdlib::Absolutepath]                 $db_tls_cert_file       = undef,
   Optional[Stdlib::Absolutepath]                 $db_tls_key_file        = undef,
   Optional[Stdlib::Absolutepath]                 $db_tls_cacert_file     = undef,
+  Optional[Hash[Enum['max_connections',
```
Comment (rwaffen) [inline manifests/init.pp:152]: complex enough to be its own custom datatype. but we can do this later, just wanted to point this out

**Review by rwaffen — APPROVED** [review]

**Comment by rwaffen** [comment]
uh, ci does not like this change :/

File: manifests/init.pp (hunk 1 above)
Comment (lbetz) [reply manifests/init.pp:152]: check

File: manifests/init.pp [hunk 2]
```diff
@@ -145,6 +149,11 @@
...
+  'max_connections_per_table',
+  'max_placeholders_per_statement',
+  'max_rows_per_transaction',
+  'wsrep_sync_wait'],Integer[1]]]                $db_options             = undef,
```
[the remark this answers is not in the document]
Comment (lbetz) [reply manifests/init.pp]: done.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**